skytemple_files/_resources/skytemple_pmdsky_debug_symbols.yml

```diff
@@ -0,0 +1,63 @@
+# Symbols that are too specific to SkyTemple or related mods to be included in pmdsky-debug.
+# Or they are values in the middle of functions.
+
+arm9:
+  functions:
+    - name: DebugPrint2
```

Comment:
What exactly is this thing? What mod adds this? When I checked the offset it seems like this is a loop within some other function...

Reply:
I might need to re-check this one. It's possible I mixed something up way back when I investigated the print functions.